subjects/java/checkpoints/breakdown-url/README.md

```diff
@@ -0,0 +1,52 @@
+## URL Parsing
+
+### Instructions
+
+Create a class `UrlParser` that provides a method to parse and validate URLs using regex. The method should extract and return url components the `protocol`, `domain`, `port`, `path` and `query` parameters. The URL is always correct.
```

Comment:
I would rephrase the second phrase like:
```
The method should extract and return the following URL components: ....
Assume the URL is always correct.
```

Reply:
yeah that's better, thanks for the suggestion.